the-then-years-bug/the-ten-years-bug.md

```diff
@@ -69,18 +69,18 @@ Now lets look at the header we attached to our recorder.js file:
 'Cache-Control': 'max-age=315360000, no-transform, public'
 ```
 
-In other words we just told the all of our end users browsers to cache our recorder for 10 years without downloading new versions, or doing any kind of communication with S3 to check for a change in the caching policy. All the minor bugs we fixed in the past still linger in our end users caches, and worse, we don't have anything to do about it. Fuck.
+We just told the all of our end users browsers to cache our recorder for 10 years without downloading new versions, or doing any kind of communication with S3 to check for a change in the caching policy. All the minor bugs we fixed in the past still linger in our end users caches, and worse, we don't have anything to do about it. Fuck.
```

Comment:
lol the first sentence is a mess. "the all of our" :)

should be "we just told all of our end users..."